fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintenanceTest.java

```diff
@@ -403,6 +434,314 @@ void flakyMerge(boolean isGrouped,
         }
     }
 
+    // Lock a directory, and commit, then try to update a record, or save a record, or do a search, and assert that nothing
```

Comment:
You are only doing a record save, and not doing the other operations.
It would be good to have a search test, as that, I think, should succeed, but saving/updating should fail.

Reply:
I would be in support of just updating the comment in this PR, and then following up with additional tests